conda/recipes/cudf/meta.yaml

```diff
@@ -90,6 +90,8 @@ requirements:
     - cubinlinker  # CUDA enhanced compatibility.
     - cuda-python >=11.7.1,<12.0a0
     {% else %}
+    # Needed by Numba for CUDA support
+    - cuda-nvcc-impl
```

Comment:
I believe that CI is currently passing because Numba is finding the system CTK. Our `rapidsai/ci` images are based on `nvidia/cuda` and that ships the pieces that Numba needs. We should have seen failures from our tests that relied on `cuda-nvcc-impl` but instead Numba located those from the system. We'd need to address https://github.com/rapidsai/ops/issues/2565 to prevent the problem of mixing system and conda versions of the CUDA Toolkit.

Reply:
Filed issue ( https://github.com/rapidsai/cudf/issues/13683 ) to follow up